openshift_tools/ircbot/openshift_sre/openshift_sre.py

```diff
@@ -345,21 +377,21 @@ def mark_channel_for_announcements(bot, trigger):
 def unmark_channel_for_announcements(bot, trigger):
```

Comment:
RE: pylint error, it's above the 30-character limit for method name.

Reply:
Ahh, thanks! Any reason to shorten it just to appease pylint?

Reply:
> Any reason to shorten it just to appease pylint?

I guess it's fine as-is. There's a bit of noise here that needs some pylint config tweaks to quiet, IMO.